### PR TITLE
bump dragmap version

### PR DESCRIPTION
**Pull request by FriederikeHanssen**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

**Comment by FriederikeHanssen** [comment]
waiting on samtools container

File: modules/nf-core/dragmap/align/main.nf [hunk 1]
```diff
@@ -2,10 +2,10 @@ process DRAGMAP_ALIGN {
     tag "$meta.id"
     label 'process_high'
 
-    conda (params.enable_conda ? "bioconda::dragmap=1.2.1 bioconda::samtools=1.15.1 conda-forge::pigz=2.3.4" : null)
+    conda (params.enable_conda ? "bioconda::dragmap=1.3.0 bioconda::samtools=1.16.1 conda-forge::pigz=2.3.4" : null)
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/mulled-v2-580d344d9d4a496cd403932da8765f9e0187774d:5ebebbc128cd624282eaa37d2c7fe01505a91a69-0':
-        'quay.io/biocontainers/mulled-v2-580d344d9d4a496cd403932da8765f9e0187774d:5ebebbc128cd624282eaa37d2c7fe01505a91a69-0' }"
+        'https://depot.galaxyproject.org/singularity/mulled-v2-mulled-v2-580d344d9d4a496cd403932da8765f9e0187774d:bb7e33c2ede38421063b8379a5fb3453645bb047-0':
```
Comment (maxulysse) [inline modules/nf-core/dragmap/align/main.nf:7]:
```suggestion
        'https://depot.galaxyproject.org/singularity/mulled-v2-580d344d9d4a496cd403932da8765f9e0187774d:bb7e33c2ede38421063b8379a5fb3453645bb047-0':
```

**Review by maxulysse — CHANGES_REQUESTED** [review]
I think there's an issue on a mulled container definition

**Comment by FriederikeHanssen** [comment]
> I think there's an issue on a mulled container definition

Thank you 🙏 I am starring at this for the past 20min, trying to figure out why it an't find it

**Comment by maxulysse** [comment]
> > I think there's an issue on a mulled container definition
> 
> Thank you pray I am starring at this for the past 20min, trying to figure out why it an't find it

Always good to have a new set of eyes

**Review by maxulysse — APPROVED** [review]
Toppen

**Review by maxulysse — APPROVED** [review]

**Review by maxulysse — APPROVED** [review]

**Comment by FriederikeHanssen** [comment]
just conda being conda

**Pull request merged**